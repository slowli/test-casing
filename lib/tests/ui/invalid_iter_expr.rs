use test_casing::test_casing;

#[test_casing(2, 5)]
fn tested_function(_arg: i32) {
    // Does nothing
}

fn main() {}
