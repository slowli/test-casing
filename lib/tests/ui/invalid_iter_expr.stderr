error[E0277]: `{integer}` is not an iterator
 --> tests/ui/invalid_iter_expr.rs:3:1
  |
3 | #[test_casing(2, 5)]
  | ^^^^^^^^^^^^^^^^^^^^ `{integer}` is not an iterator
  |
  = help: the trait `Iterator` is not implemented for `{integer}`, which is required by `{integer}: IntoIterator`
  = note: if you want to iterate between `start` until a value `end`, use the exclusive range syntax `start..end` or the inclusive range syntax `start..=end`
  = note: required for `{integer}` to implement `IntoIterator`
  = note: this error originates in the attribute macro `test_casing` (in Nightly builds, run with -Z macro-backtrace for more info)
